LethalSirenHead.csproj

```diff
@@ -95,6 +98,6 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Target Name="NetcodePatch" AfterTargets="PostBuildEvent">
-    <Exec Command="netcode-patch &quot;$(TargetPath)&quot; @(ReferencePathWithRefAssemblies->'&quot;%(Identity)&quot;', ' ')" />
+    <Exec Command="netcode-patch -nv 1.5.2 &quot;$(TargetPath)&quot; @(ReferencePathWithRefAssemblies->'&quot;%(Identity)&quot;', ' ')" />
```

Comment:
This `-nv 1.5.2` thing was also just something I found from the netcode patcher docs when I was trying to get the plugin to compile. Not sure what it does but I'm sure it won't hurt

Reply:
I'm pretty sure it stands for netcode version. It's probably the version of the netcode library to compile for.